src/appleseed.shaders/src/appleseed/as_standard_surface.osl

```diff
@@ -97,14 +97,28 @@ shader as_standard_surface
         int as_maya_attribute_hidden = 1,
         int gafferNoduleLayoutVisible = 0
     ]],
+    int in_subsurface_profile = 0
```

Comment:
Does that mean the default is now Gaussian, whereas it was Normalized Diffusion before? If so, we need to change it back to Normalized Diffusion (2).

Reply:
Ok.